pkg/vault/test/foundry/Router.t.sol

```diff
@@ -364,9 +367,57 @@ contract RouterTest is BaseVaultTest {
         uint256 bptAmountIn = bptAmountOut / 2;
         vm.prank(alice);
         uint256[] memory amountsOut = router.removeLiquidityRecovery(pool, bptAmountIn);
+        assertEq(amountsOut.length, 2, "Incorrect amounts out length");
+        assertEq(amountsOut[daiIdx], defaultAmount / 2, "Incorrect DAI amount out");
+        assertEq(amountsOut[usdcIdx], defaultAmount / 2, "Incorrect USDC amount out");
 
         BaseVaultTest.Balances memory afterBalances = getBalances(alice);
 
+        _assertBalanceChangeRemoveLiquidityRecovery(beforeBalances, afterBalances, bptAmountIn, amountsOut);
+    }
+
+    function testRemoveLiquidityRecovery__Fuzz(uint256 amountIn1, uint256 amountIn2, uint256 exitPercentage) public {
```

Comment:
With this fuzz test, do we need `testRemoveLiquidityRecovery()`?

Reply:
I'd leave it because it's easy to reason about the amounts visually. In the fuzz test everything's automatic, so the perspective is a little different.